Resources/Prototypes/_Sunrise/Entities/Objects/Fun/toys.yml

```
@@ -1115,3 +1115,123 @@
+  components:
+  - type: Sprite
+    sprite: _Sunrise/Objects/Fun/toys.rsi
+    state: plushie-seyaa
+
+- type: entity
+  id: PlushieValancius
+  parent: BasePlushie
+  name: Plush Isabella Von Valancius
+  description: A plush toy of the IAA, smelling of bureaucracy, corporate law, and cats.
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)